hawkbit-rest/hawkbit-rest-docs/src/test/java/org/eclipse/hawkbit/rest/ddi/documentation/RootControllerDocumentationTest.java

```diff
@@ -177,11 +183,15 @@ public void postCancelActionFeedback() throws Exception {
         final Long actionId = getFirstAssignedActionId(assignDistributionSet(set.getId(), target.getControllerId()));
         final Action cancelAction = deploymentManagement.cancelAction(actionId);
 
+        final DdiStatus ddiStatus = new DdiStatus(DdiStatus.ExecutionStatus.CLOSED,
```

Comment:
please adapt to use the `getJsonActionFeedback`

Reply:
`getJsonActionFeedback` is made available by the parent class `AbstractDDiApiIntegrationTest`. This parent is not available for `RootControllerDocumentationTest`. It would also not fit in a parent like `AbstractRestIntegrationTest`. Therefore I would suggest to keep it as it is.